### ISSUE TITLE
fhirpath.js support for trace

### ISSUE THREAD

**Issue by brianpos** [brianpos/fhirpath-lab#2]
This requires an update from the fhirpath.js library (researching a PR for this)

**Comment by brianpos** [comment]
This is now implemented through a fork and also now supports the proposed `join`/`split` fhirpath functions.
Once that is approved and merged into the core fhirpath.js project I'll close this issue off.
